securedrop/client.py

```diff
@@ -10,8 +10,11 @@
 from securedrop.status_packets import STATUS_PACKETS_NAME, StatusPackets
 from securedrop.login_packets import LOGIN_PACKETS_NAME, LoginPackets
 from securedrop.add_contact_packets import ADD_CONTACT_PACKETS_NAME, AddContactPackets
+from securedrop.List_Contacts_Packets import LIST_CONTACTS_PACKETS_NAME, ListContactsPackets
+from securedrop.List_Contacts_Response_Packets import LIST_CONTACTS_RESPONSE_PACKETS_NAME, ListContactsPacketsResponse
 
 DEFAULT_FILENAME = 'client.json'
+TEST_FILENAME = 'list_contacts_test.json'
```

Comment:
This variable name should be more descriptive.

Reply:
Changed to: LIST_CONTACTS_TEST_FILENAME